modules/rosa-upgrading-4_7_to_4_8_prerequisites.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * rosa_upgrading/rosa-upgrading-sts.adoc
+
+[id="rosa-upgrading-4-7-to-4-8-prerequisites_{context}"]
+= Prerequisites for upgrading from 4.7 to 4.8
+
+This section details the steps that are required before upgrading a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS) from version 4.7 to 4.8. Before upgrading, you must meet the following prerequisites:
+
+* Update the AWS Identity and Access Management (IAM) account-wide roles and policies, including the Operator policies to version 4.8.
+* After updating the roles and policies, you must update the value of the `cloudcredential.openshift.io/upgradeable-to` annotation in the `CloudCredential` custom resource to `v4.8`. This indicates that the cluster is ready to upgrade.
```

Comment:
nit: consider replacing "This" in the second sentence with "The annotation indicates..."

Reply:
I will opt to leave this as it is. The annotation itself does not indicate that the cluster is ready for the 4.8 upgrade, the value in the key-value pair does. The annotation could include the value `v4.7` and the cluster would not be ready for the upgrade. The use of the word "This" to refer to what is described in the previous sentence seems easier than explicitly stating that.